### PR TITLE
add precourse projects

### PR DESCRIPTION
**Pull request by Aitor1111**

### CODE REVIEW

File: staff/aitor-trujillo/precourse/airlines-pro/index.js [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+            console.log(flights[i])
+        }
+    }
+}
+
+// Seleccionar por id
+
+function userChoice() {
+    var choice = parseInt(
+        prompt('Introduce el ID para comprar el billete.')
```
Comment (Pauatro) [inline staff/aitor-trujillo/precourse/airlines-pro/index.js:325]: has usado let en bil let e ANDER

File: staff/aitor-trujillo/precourse/bingo-pro/index.js [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+    for (var i = 0; i < bingoCard.length; i++){
+        if (bingoCard[i].number == 'X'){
+            numbersFound++;
+        }
+    }
+    if (numbersFound == bingoCard.length){
+        turnCount = matchedNums.length;
+        alert('BINGOOOOO!');
+        console.log('BINGOOOOOOOOO!')
+        console.log('HAS COMPLETADO EL BINGO EN '+turnCount+' TURNOS.')
```
Comment (Pauatro) [inline staff/aitor-trujillo/precourse/bingo-pro/index.js:261]: otra vez? COMP LET ADO? :(

File: staff/aitor-trujillo/precourse/airlines-pro/index.js [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+    );
+    var selectFlight = flights.indexOf(
+        flights.find(x => x.id === choice)
+    );
+    if (selectFlight != -1) {
+        alert("Gracias por su compra, vuelva pronto.")
```
Comment (alejandromc23) [inline staff/aitor-trujillo/precourse/airlines-pro/index.js:331]: you have forgotten to use  ';'

File: staff/aitor-trujillo/precourse/airlines-pro/index.js [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+
+last5Flights();
+
+// FUNCIONES
+
+function askUsername() {
+    while (username.length <= 0) {
+        username = prompt('Por favor introduce tu USUARIO.');
+        if (username.length > 0) {
+            alert('Es un placer verte de nuevo ' + username + '!')
```
Comment (alejandromc23) [inline staff/aitor-trujillo/precourse/airlines-pro/index.js:55]: you have forgotten some ';'  at the end along the files

File: staff/aitor-trujillo/precourse/airlines-pro/index.js (hunk 1 above)
Comment (Aitor1111) [reply staff/aitor-trujillo/precourse/airlines-pro/index.js:325]: JAJAJAJAJAJAJAJAJAJAJAJA!

File: staff/aitor-trujillo/precourse/bingo-pro/index.js (hunk 2 above)
Comment (Aitor1111) [reply staff/aitor-trujillo/precourse/bingo-pro/index.js:261]: xDD

File: staff/aitor-trujillo/precourse/airlines-pro/index.js (hunk 3 above)
Comment (Aitor1111) [reply staff/aitor-trujillo/precourse/airlines-pro/index.js:331]: Working on that Alejandro thanks!

File: staff/aitor-trujillo/precourse/airlines-pro/index.js (hunk 1 above)
Comment (CaliZam) [reply staff/aitor-trujillo/precourse/airlines-pro/index.js:325]: jajajajajajajaja

File: staff/aitor-trujillo/precourse/airlines-pro/index.js (hunk 4 above)
Comment (Aitor1111) [reply staff/aitor-trujillo/precourse/airlines-pro/index.js]: Work finished, thanks for the review!

File: staff/aitor-trujillo/precourse/airlines-pro/index.js (hunk 3 above)
Comment (Aitor1111) [reply staff/aitor-trujillo/precourse/airlines-pro/index.js]: Finished :)

File: staff/aitor-trujillo/precourse/airlines-pro/index.js [hunk 5]
```diff
@@ -0,0 +1,319 @@
...
+
+// COSTES
+
+var costAverage = 0;
+var costSum = 0;
+
+// Usuario
+var username = "";
+
+// Busquedas
```
Comment (jordigraup) [inline staff/aitor-trujillo/precourse/airlines-pro/index.js:33]: Please correct your name, Aitor mst have an "H" : Ai-THOR!!!

**Pull request merged**